dotnet/src/Functions/Functions.OpenApi/Model/RestApiOperation.cs

```diff
@@ -257,6 +260,31 @@ private string BuildPath(string pathTemplate, IDictionary<string, object?> argum
         return pathTemplate;
     }
 
+    private object? GetArgumentForParameter(IDictionary<string, object?> arguments, RestApiParameter parameter)
+    {
+        // Try to get the parameter value by the argument name.
+        if (!string.IsNullOrEmpty(parameter.ArgumentName) &&
+            arguments.TryGetValue(parameter.ArgumentName!, out object? argument) &&
+            argument is not null)
+        {
+            return argument;
+        }
+
+        // Try to get the parameter value by the parameter name.
+        if (arguments.TryGetValue(parameter.Name, out argument) &&
+            argument is not null)
```

Comment:
Same as above

Reply:
That's how it was originally implemented, and I kept the functionality as is to avoid unintentionally introducing any breaking changes. It may be revised later to support 'nullable' parameters.